site/source/user-manual/reset-password.rst

```diff
@@ -0,0 +1,46 @@
+.. _reset-password:
+
+===================
+Reset Your Password
+===================
+
+.. contents::
+  :depth: 2
+  :local:
+
+#. Obtain the latest copy of EmbassyOS with your product key included, by `downloading it from our image server <https://images.start9.com/>`_.
+#. Flash the downloaded image to your microSD card, using `this guide <../flashing>`_.
+#. Insert the microSD card into your Embassy and power it on.
+
+    .. note:: Embassy will power up and then initialize.  Once complete, you will hear a _bep_ to indicate it is initialized and then a _chime_ sound to indicate it is online.
```

Comment:
These sounds exist on another page - initial setup - which you can reference or copy over here

Reply:
I don't see this change